### PR TITLE
dApp: Info overlay

### LINKED ISSUE #698: Context aware info or help button and page on every screen
## Description
There should be a button which leads to a pop-up/screen which explains the current screen.

@taleldayekh Could you please paste the screen, which you have showed us?

## Acceptance criteria
- There is an information icon on the top
- It is an overlay for the whole screen
- Per screen
     - One illustration
     - One paragraph
- There is one route
- It recognizes on which page it is
- The text is defined in the translation file
- It is individually formatted per screen
- The content for all the screen defined in https://github.com/raiden-network/light-client/issues/698#issuecomment-725418451 is written and implemented

## Tasks
- [ ] Explain limitations to Sahsh @taleldayekh 
- [ ] Design @sashseurat 
- [ ] Illustrations
- [ ] Implement

### PR DESCRIPTION
**Pull request by taleldayekh**
Fixes #698 

**Short description**
This PR adds the info overlay for these routes:
- Select Hub
- Open Channel
- Transfer
- Transfer (steps)
- Channels
- Raiden Account
- Withdrawal
- UDC
- Backup State

The text content for the info overlay has been added but the illustrations still remains. These will get added in a separate PR when @MariaNono has finalised them.

**Definition of Done**

- [x] Steps to manually test the change have been documented
- [x] Acceptance criteria are met
- [ ] Change has been manually tested by the reviewer (dApp) 

**Steps to manually test the change (dApp)**

1. Navigate to each of the above route and pop open the info overlay by pressing the `i` icon.

### CODE REVIEW

File: raiden-dapp/src/locales/en.json [hunk 1]
```diff
@@ -9,6 +9,54 @@
...
+    },
+    "transfer-steps": {
+      "header-image": "transfer_steps_info",
+      "header": "About Direct and Mediated Transfers",
+      "body": "A direct transfer will be possible if you have an open channel with the recipient. In any other case, the PFS will try to find the best way to route your transfer to the recipient."
+    },
+    "select-hub": {
+      "header-image": "select_hub_info",
+      "header": "About Selecting a Hub",
+      "body": "A Hub is a well connected node. You can connect to a hub or to any other node. Make sure you have enough service tokens as well as a balance of your selected token."
```
Comment (christianbrb) [inline raiden-dapp/src/locales/en.json:27]:
```suggestion
      "body": "A Hub is a well-connected node. You can connect to a hub or to any other node. Make sure you have enough service tokens to pay for the monitoring and pathfinding service as well as a balance of your selected token."
```

File: raiden-dapp/src/locales/en.json [hunk 2]
```diff
@@ -9,6 +9,54 @@
...
+    },
+    "open-channel": {
+      "header-image": "open_channel_info",
+      "header": "About Opening a Channel",
+      "body": "Opening a channel will connect you to your selected hub or node. You need to choose an amount to deposit in the channel before opening it."
```
Comment (christianbrb) [inline raiden-dapp/src/locales/en.json:32]:
```suggestion
      "body": "Opening a channel will connect you to your selected hub or node. You need to choose an amount to deposit in the channel. Later you can transfer small amounts of this deposit to any recipient in the network."   
```
Not sure, if this is well phrased

File: raiden-dapp/src/locales/en.json [hunk 3]
```diff
@@ -9,6 +9,54 @@
...
+    },
+    "channels": {
+      "header-image": "channels_info",
+      "header": "About Channels",
+      "body": "This is an overview of all your open channels for the selected token. You can choose to close and settle any channel to withdraw your tokens."
+    },
+    "account-backup": {
+      "header-image": "backup_state_info",
+      "header": "About Backing Up the State",
+      "body": "Your state is stored in the browser. You should continuously keep a backup of your state by downloading the latest json file and storing it somewhere safe."
```
Comment (christianbrb) [inline raiden-dapp/src/locales/en.json:42]:
```suggestion
      "body": "Your state is stored in the browser. You should continuously keep a backup of your state by downloading the latest JSON file and storing it somewhere safe."
```

File: raiden-dapp/src/locales/en.json [hunk 4]
```diff
@@ -9,6 +9,54 @@
...
+    },
+    "account-raiden": {
+      "header-image": "transfer_eth_info",
+      "header": "About Transferring ETH",
+      "body": "When using a Raiden account, there needs to be an ETH balance there as well. You can add a balance by transferring ETH from your main account."
```
Comment (christianbrb) [inline raiden-dapp/src/locales/en.json:47]:
```suggestion
      "body": "You need ETH on your Raiden Account to withdraw your tokens. You can add ETH by transferring it from your main account."
```

**Review by christianbrb — COMMENTED** [review]
Nice :) I have left a few proposals for changes

**Comment by taleldayekh** [comment]
Not sure what more tests to add, feel like I already covered what needs to be covered...

**Comment by weilbith** [comment]
I did a review, but I canceled it. I think we better talk about it in person when you are back from vacation. You did a cool job and I just don't want to overwrite your stuff how I think it should be so we can release it. There is no pressure for that. And I have a bunch of questions where you should get a chance to answer.

**Review by weilbith — COMMENTED** [review]
Nice pair-programming. You had some good reasons. I'm 99% happy with the final solution. :upside_down_face:

**Review by weilbith — APPROVED** [review]

**Pull request merged**